app/views/dashboards/_actions.html.erb

```diff
@@ -34,7 +34,7 @@
         href="<%= dashboard_following_tags_path %>"
         <%= params[:action] == "following_tags" ? ' aria-current="page"'.html_safe : "" %>>
         <%= t("views.dashboard.actions.following_tags") %>
-        <span class="c-indicator"><%= @user.following_tags_count %></span>
+        <span class="c-indicator"><%= @user.cached_followed_tag_names.size %></span>
```

Comment:
There were two options here and I evaluated the tradeoffs in favour of the simpler and less disruptive change because we are still testing out the feature and deciding if this is the way to go. 
 
The first option was to create two separate counter_culture fields that would be responsible for maintaining the count of followed tags and hidden tags. However, I chose to instead use the count of the the existing cached tag fields to keep track of the count.